api/DOMStringList.json

```diff
@@ -29,10 +29,10 @@
             "version_added": true
           },
           "safari": {
-            "version_added": true
+            "version_added": "5.1"
```

Comment:
The only place in WebKit where `DOMStringList` is used is `location.ancestorOrigins`, but that says it was added in Safari 6. Can you check if that data also needs updating? It would be quite strange (but not impossible) if they weren't shipped together.

Reply:
Interestingly enough, `DOMStringList` was exposed in Safari 5.1, but `location.ancestorOrigins` was introduced in Safari 6.

Reply:
That's weird, but not impossible. Getting the compile- or runtime-flags correct in every case isn't easy, stuff like this happens. (Or `DOMStringList` was used for something else I don't know about in Safari 5.1.)